pom.xml

```diff
@@ -31,9 +31,12 @@
   <name>CARDS for $PROJECT_NAME$</name>
   <description>A CARDS based project.</description>
 
-  <properties>
-    <cards.version>$CARDS_VERSION$</cards.version>
-  </properties>
+  <scm>
+    <connection>scm:git:git://github.com/data-team-uhn/$PROJECT_SHORTNAME$.git</connection>
+    <developerConnection>scm:git:git@github.com:data-team-uhn/$PROJECT_SHORTNAME$.git</developerConnection>
+    <url>https://github.com/data-team-uhn/$PROJECT_SHORTNAME$/tree/master/</url>
+    <tag>HEAD</tag>
+  </scm>
```

Comment:
This assumes that the project's source code will always be located under `github.com/data-team-uhn/` and will use a _default_ branch name of `master`.

How about, during the execution of `setup.sh`, we prompt for the project repository location and default branch name, reconfigure the local git repository to set the `origin` remote to the given remote repository location and set the local branch to the provided _default_ branch name. These settings would then be reflected in this `<scm>` part of this `pom.xml` file?

Reply:
The way this is meant to be used is:
- create a new project in github, picking the template repo as a template
- clone locally
- run the setup script
So we should already have a github repo set up when running the script. The default branch name should be different. We should update the pom to reflect the actual origin URL, and we could use the repo name in the script as an initial prompt for the `PROJECT_SHORTNAME` value.

Reply:
How about this:

- When `setup.sh` runs it will use `git remote get-url origin` to get the URL of the recently cloned git repository for the new project (as opposed to its template).
- Based on the result of `git remote get-url origin` we can provide a suggestion for the `PROJECT_SHORTNAME` value.
- The default branch name can be set by the `setup.sh` script by having it call `git rev-parse --abbrev-ref HEAD`. This way, the default branch name of this template repository will match with the default branch name of the new project repository.

Reply:
Precisely my plan!